py/objint.c

```diff
@@ -363,6 +363,10 @@ mp_int_t mp_obj_int_get_checked(mp_const_obj_t self_in) {
     return MP_OBJ_SMALL_INT_VALUE(self_in);
 }
 
+mp_uint_t mp_obj_int_get_uint_checked(mp_const_obj_t self_in) {
+    return MP_OBJ_SMALL_INT_VALUE(self_in);
```

Comment:
Reopening the discussion of https://github.com/micropython/micropython/pull/5482#discussion_r363202922 here.

Reply:
Removed this function.